packages/components/src/slot-fill/index.js

```diff
@@ -35,6 +36,10 @@ export const Slot = forwardRef( ( { bubblesVirtually, ...props }, ref ) => {
 } );
 
 export function Provider( { children, ...props } ) {
+	const parent = useContext( SlotFillContext );
+	if ( ! parent.isDefault ) {
```

Comment:
Do we need a prop in general? Wouldn't just checking for parent existence work?

Reply:
no because React actually provides us the default context that we defined when we called `createContext` even if there's no parent provider.

Also, I can't just set the default context to null because that would be a breaking change I believe.